usaspending_api/awards/models/financial_accounts_by_awards.py

```diff
@@ -148,3 +149,9 @@ class Meta:
                 "gross_outlay_amount_by_award_cpe",
             ]
         ]
+        indexes = [
+            models.Index(
+                fields=['submission','distinct_award_key', 'transaction_obligated_amount','gross_outlay_amount_by_award_cpe'],
+                name='faba_subid_awardkey_sums_idx',
+                condition=Q(disaster_emergency_fund__in=['L','M','N','O','P']))
```

Comment:
One thing we'll have to watch out for is this hard-coding. If a new set of disaster codes comes around ==> new index. Or if COVID expands to incorporate another disaster code, we'll need to update this index.